src/partial_comparable.cr

```diff
@@ -62,5 +62,16 @@ module PartialComparable(T)
     end
```

Comment:
There's `def compare_with(other : T)` above this and it's only used by this module. Seems to be a helper method so shouldn't it be private?

Reply:
This module is scheduled to be removed in `0.28.0` anyway, so don't bother (see #6611).

Reply:
This module is scheduled to be removed in 0.28.0 anyway, so don't bother (see #6611).